### ISSUE TITLE
CI tests fail against a production build

### ISSUE THREAD

**Issue by eatyourgreens** [zooniverse/front-end-monorepo#1915]
**Package**
lib-panoptes-js

**Describe the bug**
Panoptes client tests fail when `PANOPTES_ENV` is `production`.

```
lerna ERR! yarn run test:ci exited 2 in '@zooniverse/panoptes-js'
lerna ERR! yarn run test:ci stdout:
$ NODE_ENV=test mocha --reporter=min


  142 passing (1s)
  2 failing

  1) config.js
       environment from shell
         should use the value of process.env.NODE_ENV:

      AssertionError: expected 'production' to equal 'test'
      + expected - actual

      -production
      +test
      
      at Context.<anonymous> (src/config.spec.js:19:22)
      at processImmediate (internal/timers.js:439:21)

  2) config.js
       environment from shell
         should return the API host values for the test environment from the config export:

      AssertionError: expected 'https://www.zooniverse.org/api' to equal 'https://panoptes-staging.zooniverse.org/api'
      + expected - actual

      -https://www.zooniverse.org/api
      +https://panoptes-staging.zooniverse.org/api
      
      at Context.<anonymous> (src/config.spec.js:23:30)
      at processImmediate (internal/timers.js:439:21)

```

**To Reproduce**
- start a shell with a production build in Docker.
```
docker-compose run --rm bash
```
- run the CI tests in that shell
```
yarn test:ci
```

**Expected behavior**
The tests should pass.

**Additional context**
The failing test is caused by this line, which overrides `NODE_ENV` with `PANOPTES_ENV`. I can't tell if the test is wrong, or if the code is wrong.
https://github.com/zooniverse/front-end-monorepo/blob/2e21e567e9c2fcc1e778a12821f5be936c47b354/packages/lib-panoptes-js/src/config.js#L29

**Comment by eatyourgreens** [comment]
The workaround for this is to run the tests in the bash shell with
```
PANOPTES_ENV=test yarn test:ci
```

**Comment by eatyourgreens** [comment]
Fixed by #1944, I think.
